modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -34,7 +41,15 @@ class {{classname}}(object):
         }
 
 {{#vars}}
-        self._{{name}} = {{name}}
+        self._{{name}} = None
+{{/vars}}
+
+        # TODO: let required properties as mandatory parameter in the constructor.
+        #       - to check if required property is not None (e.g. by calling setter)
+        #       - ApiClient.__deserialize_model has to be adapted as well
```

Comment:
@easz thanks for the feedback. To be clear, I'm not saying we need to do it as part of this PR.  This is more for discussion on the future of the Python API client.

I've created https://github.com/swagger-api/swagger-codegen/issues/5421 for the discussion.